pytest_asdf/plugin.py

```diff
@@ -264,10 +265,13 @@ def reportinfo(self):
         return self.fspath, 0, ""
 
 
-def _parse_test_list(content):
+def _parse_test_list(content: Union[str, list]) -> dict[str, list]:
```

Comment:
I believe this needs to be typing.Dict for python 3.8
```suggestion
def _parse_test_list(content: Union[str, list]) -> Dict[str, list]:
```

Reply:
```suggestion
def _parse_test_list(content):
```
Same here